src/app/domain/deeplinks/services/DeeplinksParserService.ts

```diff
@@ -95,7 +95,17 @@ export const parseFallbackURL = (url: string | null): FallbackUrl => {
 
   try {
     const makeURL = new URL(url)
-    const fallback = makeURL.searchParams.get(FALLBACK_PARAM) ?? undefined
+    let fallback = makeURL.searchParams.get(FALLBACK_PARAM) ?? undefined
+
+    if (
+      fallback?.startsWith('cozy://') ||
+      fallback?.startsWith('https://manager.cozycloud.cc') ||
+      fallback?.startsWith('https://staging-manager.cozycloud.cc') ||
+      fallback?.startsWith('https://manager-dev.cozycloud.cc')
```

Comment:
you have `strings.cloudery.{prod|int|dev}Uri` 

Also, you should test the `mabulle://` schema, no ?